fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/StringExpNodeVisitor.java

```diff
@@ -0,0 +1,406 @@
+        result.append(value);
+        return result.toString();
+    }
+
+    @Override
+    public String literal(Double value) {
+        StringBuilder result = new StringBuilder();
+        result.append(value);
+        return result.toString();
+    }
+
+    @Override
+    public String add(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" + ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String sub(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" - ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String mult(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" * ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String div(String left, String right) {
+        StringBuilder result = new StringBuilder();
+        result.append(left);
+        result.append(" / ");
+        result.append(right);
+        return result.toString();
+    }
+
+    @Override
+    public String isNull(String expr) {
+        StringBuilder result = new StringBuilder();
+        result.append(expr);
+        result.append(" IS NULL");
+        return result.toString();
+    }
+
+    @Override
+    public String isNotNull(String expr) {
+        StringBuilder result = new StringBuilder();
+        result.append(expr);
+        result.append(" IS NOT NULL");
+        return result.toString();
+    }
+
+    @Override
+    public String between(String leftValue, String rightValue) {
+        StringBuilder result = new StringBuilder();
+        result.append(" BETWEEN ");
+        result.append(leftValue);
+        result.append(" AND ");
+        result.append(rightValue);
+        return result.toString();
+    }
+
+    @Override
+    public String column(String tableAlias, String columnName) {
+        StringBuilder result = new StringBuilder();
+        if (tableAlias != null) {
+            result.append(tableAlias);
+            result.append(".");
+        }
+        result.append(columnName);
+        return result.toString();
+    }
+
+    @Override
+    public String bindMarker(Double value) {
+        if (this.bindMarkers != null) {
+            bindMarkers.add(new DoubleBindMarkerNode(value));
+        }
+        return "?";
+    }
+
+    @Override
+    public String bindMarker(Long value) {
+        if (this.bindMarkers != null) {
+            bindMarkers.add(new LongBindMarkerNode(value));
+        }
+        return "?";
+    }
+
+    @Override
+    public String bindMarker(String value) {
+        if (this.bindMarkers != null) {
+            bindMarkers.add(new StringBindMarkerNode(value));
```

Comment:
probably need to encode in StringBindMarkerNode

Reply:
Don't need to encode values bind markers. They eliminate any chance of injection.